### PR TITLE
Additional Elixir snippets

### PR DESCRIPTION
**Pull request by radik909**
Added few Elixir snippets. 
This is my first contribution to open source projects.
So let me know if I've crossed any lines.

### CODE REVIEW

File: elixir-mode/for [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# -*- mode: snippet -*-
+# name: for
+# key: for
+# --
+for ${2:x} <- ${1:enum} do
+    $2$0
```
Comment (lpil) [inline elixir-mode/for]: Indentation should be 2 spaces by convention

File: elixir-mode/arrow [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- mode: snippet -*-
+# name: arrow (->)
+# key: a
+# --
+->
```
Comment (lpil) [inline elixir-mode/arrow]: End of file newline?

Comment (lpil) [inline elixir-mode/arrow]: Is a snippet this small useful?

File: elixir-mode/pipe [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# -*- mode: snippet -*-
+# name: pipe
+# key: p
+# --
+|>
```
Comment (lpil) [inline elixir-mode/pipe]: Is a snippet this small useful?

File: elixir-mode/for (hunk 1 above)
Comment (radik909) [inline elixir-mode/for]: Will change it.

File: elixir-mode/arrow (hunk 2 above)
Comment (radik909) [inline elixir-mode/arrow]: It was useful for me. If you and others think its not much of use, I can remove it.

File: elixir-mode/pipe (hunk 3 above)
Comment (radik909) [inline elixir-mode/pipe]: It was useful for me. If you and others think its not much of use, I can remove it.

File: elixir-mode/arrow (hunk 2 above)
Comment (AndreaCrotti) [inline elixir-mode/arrow]: yes I think it's too small to be added to the general repo..

File: elixir-mode/pipe (hunk 3 above)
Comment (AndreaCrotti) [inline elixir-mode/pipe]: yes please..

**Comment by AndreaCrotti** [comment]
Happy to merge as soon as you remove the couple of pointed out snippets and fix the indentation..

**Comment by AndreaCrotti** [comment]
Thanks @radik909 
if you could squash all the commits into one when you're done with the changes it would be amazing, and I can merge this PR then

**Comment by lpil** [comment]
@AndreaCrotti There is a "squash and merge" button in github now, so you can do it :)

**Comment by AndreaCrotti** [comment]
Oh thanks @lpil I never noticed :)

**Pull request merged**